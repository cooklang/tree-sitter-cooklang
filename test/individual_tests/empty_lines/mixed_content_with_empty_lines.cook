First step.

> A note here

Second part of first step.


New step starts here.