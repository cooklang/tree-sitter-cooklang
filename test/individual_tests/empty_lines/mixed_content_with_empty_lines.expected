(recipe 
  (step)
  (note)
  (step)
  (step))